save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -73,12 +78,15 @@ class ProjectController(
         description = "Get all projects, available for current user.",
     )
     @ApiResponse(responseCode = "200", description = "Projects successfully fetched.")
-    fun getProjects(
-        authentication: Authentication,
+    fun getProjectsWithStatus(
+        authentication: Authentication, @RequestParam status: ProjectStatus?,
     ): Flux<Project> = projectService.getProjects()
         .filter {
             projectPermissionEvaluator.hasPermission(authentication, it, Permission.READ)
         }
+        .filter { project ->
+            status?.let { project.status == it } ?: true
+        }
```

Comment:
Filtering should be done with database request

Reply:
Is it adequate to do this not at the database request level, but in the Service